examples/hacker-news/common/read-models/graphql/projection.js

```diff
@@ -0,0 +1,147 @@
+// @flow
```

Comment:
Since read-models are now based on free-style resolvers, and mandatory `graphql` is dropped, so maybe rename this read-model to more neutral naming, e.g. `default`
Most likely that `hacker-news` application is not actual. Maybe merge with dev?

Reply:
done